.github/templates/model_regression_test_results.tmpl

```diff
@@ -61,18 +61,19 @@ The print_result_core template returns data depends on available fields.
 {{- end -}}
 
 {{ define "print_table_nlu" }}
-{{- $available_types := (index .config | jsonpath `@..type`) -}}
-{{- if isKind "string" $available_types }}{{- $available_types = (index .config | jsonpath `@..type` | slice) -}}{{- end -}}
+{{- $available_types := (index .results_for_dataset | jsonpath `@..type`) -}}
+{{- if isKind "string" $available_types }}{{- $available_types = (index .results_for_dataset | jsonpath `@..type` | slice) -}}{{- end -}}
 {{- if has $available_types "nlu" -}}
 | Configuration | Intent Classification Micro F1 | Entity Recognition Micro F1 | Response Selection Micro F1 |
 |---------------|-----------------|-----------------|-------------------|
-{{ range $config_name, $config_data := .config -}}
+{{ range $config_name, $config_data_array := .results_for_dataset -}}
+{{ range $index_unused, $config_data := $config_data_array }}
```

Comment:
Am a bit lost here,  what's `index_unused`? 🙈

Reply:
is this for a mismatch in the baseline and current data?

Reply:
> Am a bit lost here, what's index_unused? 🙈

When iterating over $config_data_array, I get the index and the element. 

I will try to remove the index index as I do not need it (therefore, I seem to have named it "unused"🙈)

Reply:
💡 okay that would explain why I don't see it used anywhere

Reply:
Could you change to `unneeded`? Or can you do this: (like the first example [here](https://docs.gomplate.ca/syntax/#arrays))
```
{{ range $config_data_array }}
do something with {{ . }}...
{{ end }}
```

Reply:
The index can even be left out, changed code accordingly